docs/dev-setup/install.md

```diff
@@ -168,6 +168,21 @@ cd $GOPATH/src/github.com/hyperledger/fabric
 make peer unit-test behave
 ```
 
+### Building on Power Platform
+
+Development and build on Power (ppc64le) systems is done outside of vagrant as outlined [here](#building-outside-of-vagrant-). For ease of setting up the dev environment on Ubuntu, invoke [this script](https://github.com/hyperledger/fabric/tree/master/devenv/setupUbuntuOnPPC64le.sh) as root. This script has been validated on Ubuntu 16.04 and assumes certain things (like, development system has OS repositories in place, firewall setting etc) and in general can be improvised further.
+
+To get started on a fresh Ubuntu install
+```
+mkdir -p $HOME/git/src/github.com/hyperledger
```

Comment:
@christo4ferris As discussed, have updated the install.md file following your recommendation.